commcare_connect/opportunity/models.py

```diff
@@ -51,6 +51,12 @@ class Opportunity(BaseModel):
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs) -> None:
+        from commcare_connect.opportunity.tasks import create_learn_modules_assessments
+
+        super().save(*args, **kwargs)
+        create_learn_modules_assessments.delay(self.id)
```

Comment:
I don't think we should do this in the save function since it will get called during tests and other places.

Probably best to do it in the view.

Reply:
fixed. [f8ff725](https://github.com/dimagi/commcare-connect/pull/55/commits/f8ff7253dfadcf67cd896f20deb9b20900ed5ce8)